recipe/meta.yaml

```diff
@@ -29,10 +29,10 @@ requirements:
     - python
     - pip
     - numpy
-    - cython 0.29.33
+    - cython 0.29
     - setuptools
     - wheel
-    - packaging 22.0
```

Comment:
Doesn't seem consistent with upstream re version and location (i.e., should be in run deps?) https://github.com/yt-project/yt/blob/8a6dca3c554aebb7ccfc5a390ce0bd1148588fec/setup.cfg#L46